ref(profiling): update summary docs (#8317)
getsentry/sentry-docs: src/docs/product/profiling/index.mdx

Our bottom right side of the page has aggregated information about function regressions that Sentry may have detected during the selected time range as well as a list of the slowest functions in your application as well as some general information about the profile. We can see information like when was this profile last seen, p75, p95 and p99 values as well as the total amount of profiles we collected for this transaction.

Below the general information, we have a list of regressions that Sentry may have detected as well as a list of the slowest functions executed during this transactions.
Clicking on the before/after timings in the regressed function list will take you to example flamecharts of the before and after execution so that you can investigate what changed.

The slowest functions list serves as an overview of functions which are consuming the most time in your application - these are usually good candidates for optimization.

> The left side of the page contains am aggregate flamegraph as well as digest of the slowest and most regressed functions. We recommend reading the docs about [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) so you can understand what this data is telling us. In short, looking for wide frames that have no children is a good first thing to look at - this will allow you to identify frames that are often occupying the stack and are good candidates for optimization.

Reviewer: ```suggestion
There's an aggregate flamegraph as well as a digest of the slowest and most regressed functions on the left side of the page. As a rule of thumb, a good first thing to look for is wide frames that have no children. This will allow you to identify frames that are often occupying the stack and may be good candidates for optimization. 

We recommend reading the [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) docs to get a better understand of this data.

```

> The alternative way you can also visualize the flamegraph is by clicking on the Call Tree button which will render the flamegraph as an inverted call tree (leafmost nodes first which expand to their parent nodes). The sample column in the call tree represent how often profiling collected stack samples where that function was on the stack. Expanding of the tree can be done via click interaction on the caret before the function name (holding ⌘ while doing so will expand that entire branch all the way to the root.)

Reviewer: ```suggestion
An alternative way to visualize the flamegraph is by clicking on the "Call Tree" button. This will render the flamegraph as an inverted call tree with its leftmost nodes first, which will then expand to its parent nodes. 

The sample column in the call tree represents the frequency with which profiling collected stack samples where a function was on a stack. Tree expansion can be done via click interaction on the caret before the function name (holding ⌘ while doing so will expand that entire branch all the way to the root.)
```

I might need you to walk me through this. Having a hard time finding it in the UI. This may also really benefit from an added visual.

Using the Profiling page is typically an advanced workflow, enabling you to directly select transactions of interest and examine detailed profiling data.

## Profiling Summary Page

Selecting one of the transactions will take you to the **Profile Summary** page below. On this page, you can examine the aggregated information of this profile.

![Transaction Profile summary page](new-transaction-profile-view.png)

While the **Profiling** page can be used to view information about the profiles in our application, the **Profile Summary** page shows aggregated information from profiles collected under a single type of transaction.

This information allows you to dive deeper into what your code was executing during this transaction and may get you closer to the root of the problem.

> The top of the page has a percentile chart with profile durations and a search bar. The chart helps you visually identify possible performance regressions or changes. You can isolate the date-time ranges you want to investigate by clicking and dragging it. If you want to narrow down your investigation to profiles executed under a specific environment, use the search bar filters.

Reviewer: ```suggestion
The top of the page has a percentile chart with profile durations and a search bar. It helps you visually identify possible performance regressions or changes. You can isolate the date-time ranges you want to investigate by clicking and dragging it. If you want to narrow down your investigation to profiles executed under a specific environment, use the search bar filters.
```

The bottom right side of the page has aggregated information about function regressions detected during your selected time range plus, a list of the slowest functions. There's also a list of general information about the profile including:

- When the profile was last seen
- The total number of profiles collected for this transaction
- p75, p95, and p99 values

Below the general information is a list of regressions that Sentry may have detected. Clicking the before/after timings in the regressed function list will take you to example flamecharts of the before and after execution so that you can investigate what changed.

There's also a list of the slowest functions which are consuming the most time in your application. These are good candidates for optimization.

> 
> The left side of the page contains am aggregate flamegraph as well as digest of the slowest and most regressed functions. We recommend reading the docs about [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) so you can understand what this data is telling us. In short, looking for wide frames that have no children is a good first thing to look at - this will allow you to identify frames that are often occupying the stack and are good candidates for optimization.

Reviewer: ```suggestion

On the left side of the page is an aggregate flamegraph as well as a digest of the slowest and most regressed functions. We recommend looking for wide frames that have no children as a starting point. This will allow you to identify frames that are often occupying the stack and make good candidates for optimization. Read the [Flame Graphs and Aggregated Flame Graphs](/product/profiling/flame-charts-graphs) docs if you'd like a deeper understanding of this data. 
```

> The alternative way you can also visualize the flamegraph is by clicking on the Call Tree button which will render the flamegraph as an inverted call tree (leafmost nodes first which expand to their parent nodes). The sample column in the call tree represent how often profiling collected stack samples where that function was on the stack. Expanding of the tree can be done via click interaction on the caret before the function name (holding ⌘ while doing so will expand that entire branch all the way to the root.)

Reviewer: ```suggestion
An alternative way to visualize the flamegraph is by clicking on the "Call Tree" button which will render the flamegraph as an inverted call tree with the leftmost nodes first, which will then expand to their parent nodes. The sample column in the call tree represents the frequency with which Profiling collected stack samples where that function was on the stack. You can expand the tree by clicking on the caret before the function name. (Holding ⌘ at the same time will expand that entire branch all the way to the root.)
```
